downstreamadapter/dispatcher/dispatcher.go

```diff
@@ -106,9 +105,6 @@ type Dispatcher struct {
 	// the config of filter
 	filterConfig *eventpb.FilterConfig
 
-	// tableInfo is the latest table info of the dispatcher's corresponding table.
-	tableInfo *common.TableInfo
```

Comment:
Since batchDMLEvent has table info, we don't need to store the table info any longer. @asddongmen